engine/interpreter-dsl-test/src/test/java/org/enso/interpreter/dsl/test/ThrowableCatchTest.java

```diff
@@ -42,36 +39,19 @@ public class ThrowableCatchTest {
           CustomError::new,
           ThreadDeath::new);
 
-  private Context ctx;
-  private EnsoContext ensoCtx;
-
-  private static class CustomError extends Error {}
-
   @Before
-  public void setup() {
-    ctx =
-        Context.newBuilder()
-            .allowExperimentalOptions(true)
-            .allowIO(IOAccess.ALL)
-            .allowAllAccess(true)
-            .logHandler(System.err)
-            .option(RuntimeOptions.STRICT_ERRORS, "true")
-            .option(RuntimeOptions.LOG_LEVEL, Level.WARNING.getName())
-            .option(
-                RuntimeOptions.LANGUAGE_HOME_OVERRIDE,
-                Paths.get("../../distribution/component").toFile().getAbsolutePath())
-            .build();
-    ensoCtx = ctx.getBindings(LanguageInfo.ID).invokeMember(TopScope.LEAK_CONTEXT).asHostObject();
-    ctx.initialize(LanguageInfo.ID);
-    ctx.enter();
+  public void ctxEnter() {
+    ctxRule.context().initialize(LanguageInfo.ID);
+    ctxRule.context().enter();
   }
 
   @After
-  public void tearDown() {
-    ctx.leave();
-    ctx.close();
+  public void ctxLeave() {
+    ctxRule.context().leave();
```

Comment:
`enter()` and `leave()` should be handled automatically.

Reply:
Done in 9a36260911345b05a073502d018c6d178520f711